cleanlab/datalab/internal/spurious_correlations.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np 
+import pandas as pd
+from sklearn.model_selection import cross_val_predict, cross_val_score
+from sklearn.naive_bayes import GaussianNB
+from statistics import mode
+import warnings 
+warnings.filterwarnings('ignore')
```

Comment:
delete this line, we should not be suppressing warnings in our codebase. 

If there are warnings being printed, please explain why